packages/frontend/src/ee/features/embed/SettingsEmbed/index.tsx

```diff
@@ -203,26 +211,17 @@ const SettingsEmbed: FC<{ projectUuid: string }> = ({ projectUuid }) => {
                     </Flex>
                 </Stack>
             </SettingsGridCard>
-            <SettingsGridCard>
```

Comment:
why not Settings Grid Card?

Reply:
didnt like how the title got aligned

![CleanShot 2025-04-24 at 17 43 50@2x](https://github.com/user-attachments/assets/bcf72590-f6cf-4017-adfd-328125992b38)

![CleanShot 2025-04-24 at 17 46 10@2x](https://github.com/user-attachments/assets/f32d48a5-1dc2-4b25-8910-3b073c6058c4)